Resources/config/controller.xml

```diff
@@ -35,6 +39,10 @@
             <argument>%hwi_oauth.failed_use_referer%</argument>
             <argument>%hwi_oauth.use_referer%</argument>
 
+            <call method="setContainer">
```

Comment:
the `RedirectToServiceController` does not even have such a setter? 

Its not extending any base controller: https://github.com/hwi/HWIOAuthBundle/blob/master/Controller/RedirectToServiceController.php#L25

Reply:
👍